### PR TITLE
Master --> preview

### PR DESCRIPTION
**Pull request by Grae-Drake**
@zoelubitz as discussed.  I'll tag mentors on topics related to their work.

### CODE REVIEW

File: node/topics.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - description:  Create a server using Express and learn how Node deals with request and response objects.
 - final_description: Write your first Node server that returns different content depending on the URL.
 
-### Building APIs with Node
+### Building APIs with Node - In progress (Charlie Martin; Title TBD)
```
Comment (Grae-Drake) [inline node/topics.md]: @chasrmartin not sure if you have a working title yet; if so I can update.

File: node/topics.md [hunk 2]
```diff
@@ -29,11 +29,11 @@
 - description:  Stop the leaking memory! Clone a starter project and get garbage collection running to improve your app's performance.
 - actual_description: Find and plug the memory leaks in a bidirectional-map implementation.
 
-### Building applications with Hapi.js
+### Building applications with Hapi.js - In progress (Ben Boyle; Title TBD)
```
Comment (Grae-Drake) [inline node/topics.md]: @bboyle: let me know if you have a working title and I'll add it here.

File: node/topics.md [hunk 3]
```diff
@@ -42,11 +42,11 @@
 - description:  Error objects are first class citizens in Node and are using everywhere to handle undesirable scenarios (e.g., error first callbacks). Learn how to use them effectively, and when/where to throw them.
 - Actual description: "Create error objects to handle errors for a 'Smart TV' JavaScript API. "
 
-### Deploying Node apps
+### Deploying Node apps - In progress (Dario Craverio; Title TBD)
```
Comment (Grae-Drake) [inline node/topics.md]: @dariocravero let me know if you have a working title and I'll add it here.

File: node/topics.md [hunk 4]
```diff
@@ -62,6 +62,10 @@
 - skill_level:  Advanced
 - description:  Create the next big thing. Build a simple social network to learn the basics of Node with Express. Set up authentication, work with a Mongo database, and learn how Node works with templating engines.
 
+### Authentication with passport.js - In progress (Joe Turner; Title TBD)
```
Comment (Grae-Drake) [inline node/topics.md]: @oampo let me know if you have a working title and / or description and I'll add it here.

File: node/topics.md (hunk 2 above)
Comment (bboyle) [inline node/topics.md]: @Grae-Drake will put my thinking cap on! :)

**Pull request merged**